packages/block-editor/src/store/actions.js

```diff
@@ -232,17 +232,23 @@ export function selectBlock( clientId, initialPosition = 0 ) {
 
 /**
  * Yields action objects used in signalling that the block preceding the given
- * clientId should be selected.
+ * clientId or it's first parent from bottom to top should be selected.
```

Comment:
```suggestion
 * clientId (or optionally, its first parent from bottom to top) should be selected.
```

Nit: fix grammar and make it obvious it's an option.